### ISSUE TITLE
event hook for package is finished

### ISSUE THREAD

**Issue by StefftheEmperor2** [pyload/pyload#4246]
### Feature description
Hi, i'm trying to react on when a package is (finally) finished. the problem is, that there is no final event. i think `package_extracted` is the last event, but it won't get fired, when a package is not packed

**Comment by GammaC0de** [comment]
There are `package_finished` and `package_processed` events, have you tried them?

**Comment by StefftheEmperor2** [comment]
yeah, i think i found a workaround for myself. so no priority here, but  another question aside .. can it be that the extractarchive plugin does not get triggered, when a package is finished?

**Comment by GammaC0de** [comment]
>can it be that the extractarchive plugin does not get triggered, when a package is finished

Should not happen, unless maybe a download fails
